Client/src/store.js

```diff
@@ -14,7 +15,7 @@ const authTransform = createTransform(
   // No transformation on rehydration
   null,
   // Only applies to auth
-  { whitelist: ["auth"] }
+  { whitelist: ["auth", "ui"] }
 );
```

Comment:
Just noticed this, the whitelist should go under the persistconfig below I believe, this transform was for the image

Reply:
Good catch